src/components/BuildDirectionPicker.module.css

```
@@ -0,0 +1,68 @@
+  color: var(--ink);
+  background: var(--bg-3, rgba(255, 255, 255, 0.1));
+  border-color: var(--line, rgba(255, 255, 255, 0.12));
+}
+
+/* Stereotype = the default suggestion based on the champion's class
+   tag. Uses `currentColor` so the dashed border picks up the direction
+   pill's tint (AD red, AP blue, etc.) — visually links the dashed
+   ring to "this is the AP/AD/Tank/Supp default" without competing
+   with the solid-fill selected state. */
+.pillStereotype {
+  border-color: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)